internal/pkg/cli/storage_init.go

```diff
@@ -696,8 +697,13 @@ func (o *initStorageOpts) environmentNames() ([]string, error) {
 }
 
 func (o *initStorageOpts) RecommendedActions() []string {
-
-	newVar := template.ToSnakeCaseFunc(template.EnvVarNameFunc(o.storageName))
+	var newVar string
+	switch o.storageType {
+	case dynamoDBStorageType, s3StorageType:
+		newVar = template.ToSnakeCaseFunc(template.EnvVarNameFunc(o.storageName))
+	case rdsStorageType:
+		newVar = template.ToSnakeCaseFunc(template.EnvVarSecretFunc(o.storageName))
```

Comment:
What do you think of providing sample code to show them how to parse the secret JSON? kinda like what's in your demo. Maybe in JS as I think it's pretty common.
```
1. Update %s' code to retrieve your database secret from the env variable %s:
```js
const {username, password} = JSON.parse(process.env.%s)
\```
2. Update your code to retrieve the database endpoint from env variable %s.
```

Reply:
Yes great idea!